docs: make landing pages more informative (#524)
canonical/pebble: docs/how-to/index.md

# How-to guides

> These guides walk you through the operations lifecycle of Pebble.

Reviewer: Thanks -- I agree what we had wasn't great (it certainly isn't yet "complete" or "every step"). However, I don't really like the "operations lifecycle" jargon or really know what it means. What about something like:
```suggestion
These guides walk you through key operations you can perform with Pebble.
```

## Installation

> Installation follows a broadly similar pattern on all architectures. You can choose to install the pre-built binary or build it from the source by yourself.

Reviewer: While we're here, what about:
```suggestion
Installation follows a similar pattern on all architectures. You can choose to install the pre-built binary or build it from the source by yourself.
```
